_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -1426,58 +1459,149 @@ Set the variables for the `account-baseline-app` module in this environment in t
 .infrastructure-live/logs/_global/account-baseline/terragrunt.hcl
 [source,hcl]
 ----
+locals {
+  accounts = {
+    logs     = "409800740445"
+    security = "673123100581"
+    shared   = "384759303421"
+    dev      = "293847503945"
+    stage    = "384924092834"
+    prod     = "784260063686"
+    root     = "216044045972"
+  }
```

Comment:
So, in the LZ guide, I've moved these into the root `accounts.json` that gets included in the root `common.hcl` as `accounts = jsondecode(file("accounts.json"))`. Then every `terragrunt.hcl` that needs it does

```
locals {
  # A local for more convenient access to the accounts map.
  accounts = local.common_vars.locals.accounts
  ...
}
```

It's a nit, but wonder if it's something you want to follow here.